### ISSUE TITLE
Fix bug "Cannot read properties of undefined" and its friends

### ISSUE THREAD

**Issue by amclark42** [amclark42/whova-session-streamlined#2]
Because the script errors out, it is not able to modify a subsequent navigation to a session. I suspect the script needs to do a check on whether the current page is a session or not.

The console error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'classList')
    at onLoad (VM342 userscript.html:81:16)
    at Proxy.<anonymous> (VM342 userscript.html:171:31)
    at St (<anonymous>:9:89)
    at t.<computed> (<anonymous>:59:418)
    at St (<anonymous>:9:89)
    at r (<anonymous>:22:484)
    at <anonymous>:23:112
    at <anonymous>:69:256
    at g (<anonymous>:69:275)
```

**Comment by amclark42** [comment]
Opening this back up because this error is still popping up elsewhere in the code. When it happens, the toggle buttons in the session sidebar disappear. The left-hand sidebar is generally okay.

I'm currently thinking that the mutation observer might need to do more work to make sure the time is right for modifying the page.

**Comment by amclark42** [comment]
Another idea: integrate try/catch structures into the modification parts of the code.

**Comment by amclark42** [comment]
I've fixed the session nav in cec72e9 (v0.7), but I'm not closing this issue yet. I'd like to integrate more testing inside `updateSessionNav()`.

**Comment by amclark42** [comment]
Closed with 8e4eb3b. 😄

**Comment by amclark42** [comment]
Closed with 8e4eb3b. 😄
